src/events/api/views.py

```diff
@@ -8,3 +10,18 @@ class TalkDetailAPIView(RetrieveAPIView):
 
     queryset = models.TalkProposal.objects.all()
     serializer_class = serializers.TalkDetailSerializer
+
+
+class TutorialListAPIView(views.APIView):
+    def get(self, request):
+        tutorial_data = TutorialProposal.objects.all()
```

Comment:
Use ProposedTutorialEvent, or you should check tutorial is accepted.
(TutorialProposal include accepted tutorial and unaccepted, but ProposedTutorialEvent only has accepted tutorial.)

Reply:
thanks! I've modified my code. Please help check